index.js

```diff
@@ -1 +1,26 @@
-// Write your solution in this file!
+var customerName = 'bob';
+var bestCustomer;
+const leastFavoriteCustomer = 'Nihar';
+
+function upperCaseCustomerName() {
+    customerName = customerName.toUpperCase();
+    return customerName;
+}
+
+function setBestCustomer() {
+    bestCustomer = 'not bob';
+    
+    return bestCustomer;
+}
+
+function overwriteBestCustomer() {
+    bestCustomer = 'maybe bob';
+
+    return bestCustomer;
+}
+
+function changeLeastFavoriteCustomer() {
+    leastFavoriteCustomer = 'Steve';
```

Comment:
Wow man...

Reply:
Oh it's another Steve he goes to a different code academy